code/game/objects/structures/roguetown/newtree.dm

```diff
@@ -184,7 +187,10 @@
 	if(istype(target, /turf/open/transparent/openspace))
 		var/obj/structure/flora/newtree/T = new(target)
 		T.base_state = "center-leaf[rand(1,2)]"
+		T.tree_type = src.tree_type
 		T.update_icon()
+		T.build_branches()
```

Comment:
don't do this... there is a reason its in a subsystem

Reply:
will revert to subsystem, but im curious why

Reply:
few reasons main is so that spawned trees via other means would generate trees, also its super intensive on init

Reply:
Reverted to SS